src/message.js

```diff
@@ -181,23 +182,35 @@ Message.prototype.decryptSessionKeys = async function(privateKeys, passwords) {
       throw new Error('No public key encrypted session key packet found.');
     }
     await Promise.all(pkESKeyPacketlist.map(async function(keyPacket) {
-      const privateKeyPackets = new packet.List();
-      privateKeys.forEach(privateKey => {
-        privateKeyPackets.concat(privateKey.getKeys(keyPacket.publicKeyId).map(key => key.keyPacket));
-      });
-      await Promise.all(privateKeyPackets.map(async function(privateKeyPacket) {
-        if (!privateKeyPacket) {
-         return;
-        }
-        if (!privateKeyPacket.isDecrypted()) {
-          throw new Error('Private key is not decrypted.');
-        }
-        try {
-          await keyPacket.decrypt(privateKeyPacket);
-          keyPackets.push(keyPacket);
-        } catch (err) {
-          util.print_debug_error(err);
+      await Promise.all(privateKeys.map(async function(privateKey) {
+        const primaryUser = await privateKey.getPrimaryUser(/* symEncryptedPacket.created, userId */);
```

Comment:
What does this comment mean?

Reply:
Why can't we at least pass in the time though?

Reply:
Yeah we can do that

Reply:
Actually, it turns out the `symEncryptedPacket.created` property was wishful thinking, and it doesn't exist, so we don't have the time here.